packages/shared/lib/contexts/governance/stores/participation-overviews.store.ts

```diff
@@ -0,0 +1,29 @@
+import { derived, get, Readable, writable } from 'svelte/store'
+import type { ParticipationOverview } from '@iota/wallet/out/types'
+import { selectedAccountIndex } from '@core/account/stores'
+import { getParticipationOverview } from '@core/account/api/getParticipationOverview'
+import { DEFAULT_PARTICIPATION_OVERVIEW } from '../constants'
+
+export const allParticipationOverviews = writable<{ [accountId: number]: ParticipationOverview }>({})
+
+export const participationOverviewForSelectedAccount: Readable<ParticipationOverview> = derived(
+    [selectedAccountIndex, allParticipationOverviews],
+    ([$selectedAccountIndex, $allParticipationOverviews]) => {
+        if ($selectedAccountIndex >= 0) {
+            return (
+                $allParticipationOverviews[$selectedAccountIndex] ??
+                ({ participations: [] } as unknown as ParticipationOverview)
+            )
+        } else {
+            return { participations: [] } as unknown as ParticipationOverview
```

Comment:
We can use `DEFAULT_PARTICIPATION_OVERVIEW` here rather than having to set / cast it each time, although we need to modify the constant just a bit.